### ISSUE TITLE
Give a different name to Time locked tokens to make them easier to spot.

### ISSUE THREAD

**Issue by individualpirate** [fsn-dev/Document#2]
Renaming suggestion for time locked tokens. In order to clearly distinguish time locked backend FSN time locked frontend FSN and regular FSN I would suggest calling them different things and clearly listing what each is capable of. Would also suggest splitting them into different token categories on the swap market.

FSN
Can be sent to exchanges
Can be split into FFSN and BFSN between two wallets
Can be traded on Swap market
Can be sent between wallets

FFSN (time locked frontend FSN)
The start time of the time lock of FFSN can be in the past, right now, or it can be in the future. The end time is in the future, but not at infinity
Is lost at the end of the time-lock
If start time of time-lock is not in the future 5000 FFSN and the end time is more than 30 days away it can be staked to run a node and purchase tickets to receive rewards as signing node
Can be traded on Swap market
Can be sent to a staking pool, which promises to distribute staking rewards
Can be used to convert BFSN into FSN if the time lock ends at the same time or after the time connected to the BFSN
Can be sent between wallets, but not accepted by exchanges
As FFSN will eventually disappear a holder of FFSN should not be regarded as a holder of FSN

BFSN (Backend time locked FSN)
The start time of the time lock of BFSN is in the future and the end time of the time lock in is at infinity.
Is converted to FSN at the start time of the timelock
Can be traded on Swap market
Can be converted into FSN by use of FFSN if the time lock ends at the same time or after the time connected to the BFSN
Can be sent between wallets, but not accepted by exchanges
As BFSN will eventually become FSN a holder of BFSN should be regarded as a holder of FSN

**Comment by zhaojun-sh** [comment]
Good idea. Can you write an article about timelock? The FOSC community now has this reward program. https://medium.com/@fusionprotocol/2-million-fsn-open-source-campaign-progress-report-november-12-b92e0cb4848f?source=---------3------------------

**Comment by individualpirate** [comment]
I would be very happy to do so, but perhaps I should hold off until an update is made in their presentation? It'd make for a better article if written after such a change.
